x-pack/plugins/observability/public/pages/slos/components/badges/slo_rules_badge.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiButtonIcon, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { Rule } from '@kbn/triggers-actions-ui-plugin/public';
+import { SloRule } from '../../../../hooks/slo/use_fetch_rules_for_slo';
+
+interface Props {
+  rules: Array<Rule<SloRule>> | undefined;
+  onClick: () => void;
+}
+
+export function SloRulesBadge({ rules, onClick }: Props) {
+  return rules === undefined || rules.length > 0 ? null : (
+    <EuiToolTip
+      position="top"
+      content={i18n.translate('xpack.observability.slo.slo.rulesBadge.label', {
+        defaultMessage:
+          'There are no rules configured for this SLO yet. You will not receive alerts when SLO is breached.',
+      })}
+      display="block"
+    >
+      <EuiButtonIcon iconType="warning" color="danger" onClick={onClick} />
```

Comment:
One thing I've noticed is that the button size is 24px by 24px, which is 4px more than the badges. Visible only when hover:
<img width="477" alt="image" src="https://user-images.githubusercontent.com/1376800/232906955-146a11a1-e6d7-4404-b2b9-d4e8e3460390.png">

Maybe we can let it slide if it's too annoying to fix.

Reply:
Turned it into a `EuiBadge` so it's the same size as the others. Can't make the hover happen on a badge though, which is a shame, but oh well.